jill/utils/version_utils.py

```diff
@@ -71,131 +73,66 @@ def get_version(version: str):
 
 def read_releases(stable_only=False) -> List[Tuple]:
     """
-    read release info from local storage
+    read release info from versions.json (VERSIONS_URL)
     """
-    # TODO: read from versions.json (#16)
-    cfg_file = RELEASE_CONFIGFILE
-    if not os.path.isfile(cfg_file):
-        return []
-    with open(cfg_file, "r") as csvfile:
-        releases = []
-        for row in csv.reader(csvfile):
-            releases.append(tuple(row))
-    if stable_only:
-        releases = list(filter(lambda x: x[0] != "latest",
-                               releases))
+    print(f'{color.GREEN}querying release information (from julialang-s3.julialang.org){color.END}')
+    v = json.loads(requests.get(VERSIONS_URL).text)
```

Comment:
TODO:

there's an associated schema file `versions-schema.json`, we can use `jsonschema` to make sure that the content is valid.

https://python-jsonschema.readthedocs.io/en/stable/